test/fixtures/good.toml

```diff
@@ -0,0 +1,33 @@
+# This is a TOML document.
```

Comment:
Should there be a bad.toml to go with this good.toml that will get run by test_cli.go?

Reply:
Will add. I may use this opportunity to restructure the fixture directory structure as well. There isn't any organization right now.

Reply:
Added a bad.toml file - will save the refactor for later so these changes can get pushed